diff -aur target-org/webkit/Source/JavaScriptCore/runtime/Options.h target/webkit/Source/JavaScriptCore/runtime/Options.h
--- target-org/webkit/Source/JavaScriptCore/runtime/Options.h	2020-11-02 14:19:51.356658611 +0100
+++ target/webkit/Source/JavaScriptCore/runtime/Options.h	2020-11-02 15:02:05.980526151 +0100
@@ -393,7 +393,7 @@
     v(bool, gcAtEnd, false, Normal, "If true, the jsc CLI will do a GC before exiting") \
     v(bool, forceGCSlowPaths, false, Normal, "If true, we will force all JIT fast allocations down their slow paths.") \
     v(unsigned, gcMaxHeapSize, 0, Normal, nullptr) \
-    v(unsigned, forceRAMSize, 0, Normal, nullptr) \
+    v(unsigned, forceRAMSize, 536870912, Normal, nullptr) \
     v(bool, recordGCPauseTimes, false, Normal, nullptr) \
     v(bool, logHeapStatisticsAtExit, false, Normal, nullptr) \
     v(bool, forceCodeBlockToJettisonDueToOldAge, false, Normal, "If true, this means that anytime we can jettison a CodeBlock due to old age, we do.") \
